README.md

```diff
@@ -56,6 +56,7 @@ You can generate messages between nodes by querying a distributed database Mnesi
 
 To setup a Mnesia cluster, start several Erlang nodes with unique names e.g. `a@`, `b@`, `c@`, etc. and start the database on all of them:
 ```
+$ ./erlangpl --node a@127.0.0.1 --cookie mnesia_cluster
```

Comment:
Please remove this line, because if you try to start the ./erlangpl script before starting the a@ node, it fails with `ERROR: failed to connect to remote node 'a@127.0.0.1'`